core/services/s4/postgres_orm.go

```diff
@@ -0,0 +1,95 @@
+package s4
+
+import (
+	"database/sql"
+	"fmt"
+	"time"
+
+	"github.com/smartcontractkit/chainlink/v2/core/logger"
+	"github.com/smartcontractkit/chainlink/v2/core/services/pg"
+	"github.com/smartcontractkit/chainlink/v2/core/utils"
+
+	"github.com/pkg/errors"
+	"github.com/smartcontractkit/sqlx"
+)
+
+type postgres struct {
```

Comment:
maybe "postgresORM"? It looks like most modules in core call this object just "orm".